index.js

```diff
@@ -67,7 +67,13 @@ function PinoColada () {
     if (contentLength != null) output.push(formatBundleSize(contentLength))
     if (responseTime != null) output.push(formatLoadTime(responseTime))
 
-    return output.filter(noEmpty).join(' ')
+    var str = output.filter(noEmpty).join(' ')
+
+    // if error, output stack trace on new line
+    var err = obj.err
+    if (err && err.stack) str += nl + err.stack
```

Comment:
okay sweet. added some more commits: reverted my previous code, used your suggestion (with a minor change), added a way to handle the errors from `pino-http` (when the error is not `obj` but `obj.error`).